tests/integration/deploy/test_deploy_command.py

```diff
@@ -1198,6 +1209,33 @@ def test_deploy_logs_warning_with_cdk_project(self):
         self.assertIn(warning_message, deploy_process_execute.stdout)
         self.assertEqual(deploy_process_execute.process.returncode, 0)
 
+    def _assert_companion_stack(self, cfn_client, companion_stack_name):
+        try:
+            cfn_client.describe_stacks(StackName=companion_stack_name)
+        except ClientError:
+            self.fail("No companion stack found.")
+
+    def _assert_companion_stack_content(self, ecr_client, companion_stack_name):
+        stack = boto3.resource("cloudformation").Stack(companion_stack_name)
+        resources = stack.resource_summaries.all()
+        for resource in resources:
+            if resource.resource_type == "AWS::ECR::Repository":
+                policy = ecr_client.get_repository_policy(repositoryName=resource.physical_resource_id)
+                self._assert_ecr_lambda_policy(policy)
+            else:
+                self.fail("Non ECR Repo resource found in companion stack")
+
+    def _assert_ecr_lambda_policy(self, policy):
+        policy = json.loads(policy.get("policyText"))
+        statements = policy.get("Statement")
+        self.assertEqual(len(statements), 1)
+        lambda_policy = statements[0]
+        self.assertEqual(lambda_policy.get("Principal"), {"Service": "lambda.amazonaws.com"})
+        actions = lambda_policy.get("Action")
+        self.assertEqual(
+            sorted(actions), sorted(["ecr:GetDownloadUrlForLayer", "ecr:GetRepositoryPolicy", "ecr:BatchGetImage"])
```

Comment:
["ecr:GetDownloadUrlForLayer", "ecr:GetRepositoryPolicy", "ecr:BatchGetImage"] is this list stored somewhere in our codebase? Would be nice if this is an import and any changes to our ECR policies won't need an update on this test

Reply:
It's hard coded in the template generation part.
Not importing in the test as this test acts as a sanity check for any changes to the ECR policy.